integrations/google_genai/src/haystack_integrations/components/generators/google_genai/chat/chat_generator.py

```diff
@@ -265,6 +357,50 @@ class GoogleGenAIChatGenerator:
     This component provides an interface to Google's Gemini models through the new google-genai SDK,
     supporting models like gemini-2.0-flash and other Gemini variants.
 
+    ### Thinking Support (Gemini 2.5 Series)
```

Comment:
This intro to thinking seems a bit too long and detailed for docstrings. I would convert it into documentation.
WDYT?

Reply:
Agreed